docs/src/lang/booleans.md

```diff
@@ -235,6 +244,8 @@ not False   # True
 **Arguments:** Two arguments. Although they can be arbitrary expressions, the
 result is only defined when both arguments are evaluated to Boolean values.
 
+**Apalache type:** `(Bool, Bool) => Bool`
```

Comment:
Probably need to a note clearly stating that this `=>` is not the same as the language-level `=>`.  Something like:

```suggestion
**Apalache type:** `(Bool, Bool) => Bool`. Note that the `=>` operator at the type level expresses the relation of inputs types to output types for operators, and as opposed to the `=>` expressing the implication relation at the value level.
```

Reply:
Amazing. I did not notice it at all :)